gcloud/storage/_helpers.py

```diff
@@ -187,3 +190,14 @@ def _setter(self, value):
         self._patch_properties({fieldname: value})
 
     return property(_getter, _setter)
+
+
+def _base64_md5hash(bytes_to_sign):
```

Comment:
Yes. I added all the concerns about buffer size after submitting the PR.

For the now this was just a relocation of the helper from regression tests.

Will update and ping you.